### PR TITLE
Cpni to npsa rebranding

### PR DESCRIPTION
**Pull request by DLEEMOJ**
Requested by National Protective Security Authority (NPSA) to update a link on our system lockdown and hardening site that has mentions CPNI and old link to NPSA.

Could you review the PR request @L-Crosby please?

### CODE REVIEW

**Comment by DLEEMOJ** [comment]
Hi Jon, this pull request to update the branding name NPSA at their request. Could you review this?

File: dita/system-lockdown-and-hardening-standard.dita [hunk 1]
```diff
@@ -286,11 +286,11 @@
     <body>
       <p>The following external reference sources provide a good source of information on IT system hardening and secure system configuration.</p>
     </body>
-    <topic id="cpni">
-      <title>CPNI</title>
+    <topic id="nspa">
+      <title>NSPA</title>
```
Comment (cybersquirrel) [inline dita/system-lockdown-and-hardening-standard.dita:290]: Should we expand the abbreviation on first use here?

**Review by cybersquirrel — APPROVED** [review]
Suggest expanding the definition of NPSA on first use, but otherwise content

**Comment by MoJEProsser** [comment]
@cybersquirrel I've updated this branch to fix all merge conflicts, bring everything up to date, and expanded the NPSA acronym on first use.

**Review by cybersquirrel — APPROVED** [review]
Looks good to me - thank you.

**Pull request merged**